open-api.yaml

```diff
@@ -1850,6 +1910,66 @@ paths:
         - $ref: '#/components/parameters/Content-Type'
     parameters:
       - $ref: '#/components/parameters/indexUid'
+  '/indexes/{indexUid}/documents/delete':
+    post:
+      operationId: indexes.documents.remove
+      summary: Delete documents
+      description: Delete a selection of documents based on a filter.
+      tags:
+        - Documents
+      security:
+        - apiKey: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                offset:
+                  type: number
+                  description: Number of documents to skip.
+                  default: 0
+                limit:
+                  type: number
+                  description: Maximum number of documents returned.
+                  default: 20
+                fields:
+                  type: array
+                  description: 'Array of attributes whose fields will be present in the returned documents. By default all attributes will be returned.'
```

Comment:
Is this really a part of this route?

Reply:
Hey, no, this is part of the fetch route introduced in https://github.com/meilisearch/specifications/pull/234.
I should not have made two PRs that highly depend on each other; now I highly regret this choice :smiling_face_with_tear: